index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <title>Beryl's Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100;500&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100;500&display=swap" rel="stylesheet">
+
+</head>
+<body>
+<main>
+
+  <header>
+    <nav>
+      <ul class="logo" >
+        <li><a href=""> <h3 class="gakii"> Gakii Beryl</h3></a></li>
```

Comment:
**[OPTIONAL]** - Although this requirement is optional but strongly recommended 🙏👍

Your Design
![image](https://user-images.githubusercontent.com/78251927/226973919-4e088cb7-a420-4cdc-9520-e48aac9f68f8.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/226973799-9be98b3e-df1e-4900-8e9b-4bb088fdf69e.png)

According to the project requirements, It is important to make our website look exactly like the design as possible(e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

That will thus show our Front-end skill ability and attention to detail in following design guidelines closely 💪.

Kindly let's consider using the same font as the template on Figma. 👍